hudi-utilities/src/main/java/org/apache/hudi/utilities/transform/Transformer.java

```diff
@@ -45,4 +46,8 @@ public interface Transformer {
    */
   @PublicAPIMethod(maturity = ApiMaturityLevel.STABLE)
   Dataset<Row> apply(JavaSparkContext jsc, SparkSession sparkSession, Dataset<Row> rowDataset, TypedProperties properties);
+
+  default Schema schemaTransform(JavaSparkContext jsc, SparkSession sparkSession, Schema incomingSchema, TypedProperties properties) {
```

Comment:
rename: trasnformedSchema() to indicate its returning the transformed schema.

Reply:
Addressed